mac/Keyman4MacIM/write-download_info.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+f#!/bin/sh
```

Comment:
looks like a stray `f` character on the first line

Reply:
Thanks. How can I not see something like that?